api/graphql/query/CalendarEventInvite/availability.js

```diff
@@ -1,14 +1,13 @@
-import User from 'Services/User';
 import Models from 'Models';
 
-const availability = async (parent) => {
+const availability = async (parent, ags, ctx) => {
```

Comment:
destruct loaders from ctx.

Reply:
My doubt is super stupid. To destruct loaders from ctx, I have to do ctx.loaders.destroy right?
and save it in a different variable before destructing it and use it to call load?

Reply:
Learn js: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
Oh I am so sorry. I did not know this is what you meant by destruct :P